python/doc/examples/reliability_sensitivity/reliability/plot_form_explained.py

```diff
@@ -273,194 +324,238 @@
+graph_standard_space.setLegendPosition("lower right")
 cc = ot.Curve(
-    [0.0, designPointStandardSpace[0]],
-    [0.0, designPointStandardSpace[1]],
+    [0.0, design_point_standard_space[0]],
+    [0.0, design_point_standard_space[1]],
     r"$\beta_{HL}$ distance",
 )
 cc.setLineStyle("dashed")
 cc.setColor("black")
-graphStandardSpace.add(cc)
-view = otv.View(graphStandardSpace)
+graph_standard_space.add(cc)
+graph_standard_space.setLegendPosition('topright')
+view = otv.View(graph_standard_space, square_axes=True)
 
 # %%
-# Estimating the failure probability integral
-# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# The FORM approximation
+# ^^^^^^^^^^^^^^^^^^^^^^
 #
-# The last step of the FORM algorithm is to replace the domain of integration by the half-space at
-# the design point. In this simple example the half-space is delimited by the tangent at the design
-# point in the standard space.
+# The last step of the FORM algorithm is to replace the failure domain boundary by the hyperplane
+# which is tangent to the failure domain at the design point in the standard space.
+# To draw this hyperplane :math:`\mathcal{P}_{\vect{z}^*}`, we define the function from
+# :math:`\Rset^2` to :math:`\Rset` defined by:
 #
-# The expression of the failure domain boundary in the standard space is the composition of the
-# hyperbole :math:`h:x \mapsto 10/x` and the inverse transform on the first variable. We can compute
-# the gradient (here the first derivative of a 1D function :math:`h(u_0)` ) at any given point with the
-# getGradient method :
-
-# %%
-u0 = [designPointStandardSpace[0]]
-du0 = failureBoundaryStandardSpace.getGradient().gradient(u0)
-print("abscissa of the design point u0  = ", u0[0])
-print("value of the failure boundary at u0 = ", failureBoundaryStandardSpace(u0)[0])
-print("value of the gradient of the failure boundary at u0 = ", du0[0, 0])
-
-
-# %%
-# In the standard space the equation of the tangent :math:`\Pi_{u_0}(x)` is given by
+# .. math::
+#
+#    M \rightarrow \scalarproduct{\nabla \widetilde{\model}(\vect{z}^*)}{\vect{Z^*M}}
+#
+# where :math:`\nabla \vect{\widetilde{\model}(\vect{z}^*)}` is the gradient of the
+# function :math:`\widetilde{\model}`
+# at the design point :math:`Z^*(\vect{z}^*)`.
+# Then, the tangent hyperplane is the isoline associated to the zero level of the previous function :
 #
 # .. math::
 #
-#    \Pi_{u_0}(x) = (h \circ T^{-1}) (u_0) + \frac{d}{dx} (h \circ T^{-1}) (u_0) (x-u_0)
+#    \mathcal{P}_{z^*} = \{ \vect{z} \in \Rset^2 \, | \,
+#                \scalarproduct{\nabla \widetilde{\model}(\vect{z}^*)}{\vect{Z^*M}} = 0\}
 #
-x = np.linspace(1.1, 5.0, 100)
-hyperplane = failureBoundaryStandardSpace(u0)[0] + du0[0, 0] * (x - u0)
-curveHyperplane = ot.Curve(x, hyperplane, r"$\Pi_{u_0}$ (FORM)")
-curveHyperplane.setLineStyle("dashed")
-curveHyperplane.setColor("green")
-graphStandardSpace.add(curveHyperplane)
-view = otv.View(graphStandardSpace)
+# We can use the class LinearFunction.
```

Comment:
This should point to the API help page of `LinearFunction`.

Reply:
yes